app/views/email_authentications/passwords/edit.html.erb

```diff
@@ -1,9 +1,10 @@
-<h1><%= t("devise.password_creation.reset_header") %></h1>
+<% unless resource.reset_password_period_valid? == false %>
```

Comment:
Consider `<% if resource.reset_password_period_valid? %>`

Reply:
Yeah. I wanted to keep the main case first. But I do agree with you that unless/else is poor form.